lib/SGN/Controller/AJAX/BrAPI.pm

```diff
@@ -589,9 +589,23 @@ sub observation_levels_GET {
 sub seasons : Chained('brapi') PathPart('seasons') Args(0) : ActionClass('REST') { }
 
 sub seasons_POST {
-    my $self = shift;
-    my $c = shift;
-    seasons_process($self, $c);
+	my $self = shift;
+	my $c = shift;
+	my $clean_inputs = $c->stash->{clean_inputs};
+	my $postedData = $clean_inputs;
```

Comment:
Probably doesn't really matter since this is a mock thing but it may be more with convention to put processing code in one of the brapi modules and call that from here.

Reply:
Chatted with Nick about this.  We agreed that it probably makes more sense to not follow the standard convention because the implementation for this POST endpoint isn't actually saving anything to the database, and it will (hopefully) be obvious to anyone looking at this code in the future that this is the case.